### PR TITLE
Add support for sqrt und pow operators

### PR DESCRIPTION
**Pull request by maltebaumann**
Hey, I needed support for `tf.sqrt` and `tf.pow` during conversion. So I added both operations and updated the mapping and readme. My conversion seems to work fine now. If I missed anything, please let me know and I'll update accordingly :)

### CODE REVIEW

File: tfcoreml/_layers.py [hunk 1]
```diff
@@ -860,6 +860,24 @@ def square(op, context):
...
       output_name, [input_name, input_name], output_name, 'MULTIPLY')
 
+def pow(op, context):
+
+  input_name = compat.as_bytes(op.inputs[0].name)
+  output_name = compat.as_bytes(op.outputs[0].name)
+
+  context.translated[output_name] = True
+  context.builder.add_unary(
+    output_name, input_name, output_name, 'power')
```
Comment (aseemw) [inline tfcoreml/_layers.py]: I guess you also want to read the value of alpha from TF and set it for CoreML (i.e. y = x^alpha).

**Comment by aseemw** [comment]
Thank you!! 
Looks good. I made one comment for the pow layer. Also it would be great if you could add two unit tests for these ops.

File: tfcoreml/_layers.py (hunk 1 above)
Comment (maltebaumann) [reply tfcoreml/_layers.py]: Oh I've got no idea how I missed that. Will fix and add tests. Thanks!

**Comment by aseemw** [comment]
See PR#137

**Pull request closed without merging**